pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -197,4 +211,8 @@ stdenv.mkDerivation {
        collision due to the ld/as wrappers/symlinks in the latter. */
     priority = 10;
   };
-}
+} // lib.optionalAttrs stdenv.targetPlatform.isXtensa {
+  postPatch = ''
+    cp -RT ${espressifXtensaOverlays stdenv.targetPlatform}/binutils .
+  '';
+})
```

Comment:
I think we should inline this even if it is causing rebuilds.

Reply:
Plan is to submit a separate PR targeting staging (see --disable-gdb TODO above)

Reply:
Inlined to `prePatch`.